week-01/assessment/capsuleHotel/src/Hotel.java

```diff
@@ -0,0 +1,142 @@
+        boolean exitConfirmation = false;
+        while (operation != 4 || !exitConfirmation) {
+            switch (operation) {
+                case 1:
+                    boolean hasSuccessfullyCheckedIn = checkIn(console, guestList);
+                    while (!hasSuccessfullyCheckedIn) {
+                        hasSuccessfullyCheckedIn = checkIn(console, guestList);
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+                case 2:
+                    boolean hasSuccessfullyCheckedOut = checkOut(console, guestList);
+                    while (!hasSuccessfullyCheckedOut) {
+                        hasSuccessfullyCheckedOut = checkOut(console, guestList);
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+
+                case 3:
+                    view(console, guestList, capsulesCount);
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+
+                case 4:
+                    exitConfirmation = exit(console);
+                    if (exitConfirmation) {
+                        break;
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+            }
+        }
+
+    }
+
+    //CHECK IN
+    public static boolean checkIn(Scanner console, String[] guestList) {
+        System.out.println("What is your name? ");
+        String name = console.next();
+
+        System.out.println("What is your capsule number? ");
+
+        int capsuleNumber = console.nextInt() - 1;
+        if (guestList[capsuleNumber] == null) {
+            guestList[capsuleNumber] = name;
+            return true;
+        } else {
+            System.out.println("This capsule is occupied. ");
+            return false;
+        }
+    }
+
+    //CHECK OUT
+
+    public static boolean checkOut(Scanner console, String[] guestList) {
+
+        System.out.println("What is your capsule number? ");
+        int capsuleNumber = console.nextInt() - 1;
+
+        if (guestList[capsuleNumber] == null) {
+            System.out.println("There is no one occupying that room. ");
+            return false;
+        } else {
+            guestList[capsuleNumber] = null;
+            System.out.println("Thank you for staying at the Capsule Hotel! You have successfully checked out. ");
+            return true;
+        }
+    }
+
+    //VIEW
+    public static void view(Scanner console, String[] guestList, int capsulesCount) {
```

Comment:
Great job with your view method - really impressed you got this working independently this weekend!